src/Integration/Binding/BindingController.cs

```diff
@@ -145,7 +146,7 @@ void IBindingWorkflowExecutor.BindProject(BindCommandArgs bindingArgs)
                     Strings.Bind_FirstTimeBinding :
                     Strings.Bind_UpdatingNewStyleBinding);
 
-            return new NewBindingWorkflow(host, bindingArgs, configProvider);
+            return new BindingWorkflow(host, bindingArgs, new NoOpNuGetBindingOperation(host.Logger));
```

Comment:
Instead of using a completely separate binding workflow class, we now use the same binding class but with different behaviours plugged in (I'll delete the redundant new binding workflow code separately).
The next step is to plug in a "solution binding operation" that saves the data to a different location and doesn't add the files to the solution folder.